src/RISCV.hs

```diff
@@ -74,6 +76,7 @@ import RISCV.RV32_M
 import RISCV.RV32_A
 import RISCV.RV32_F
 import RISCV.RV32_D
+import RISCV.RV32_B hiding (rori, bclri, bexti, binvi, bseti)
```

Comment:
Why are you hiding these 5 instructions?

Reply:
Strictly, the encodings for those differ between RV32 and RV64, with the former requiring the latter's `shamt[5]` bit be `0`.  But for TestRIG's purposes, that surely doesn't matter that much, so the default exports from `RISCV` are the RV64 forms.